templates/raspbian_cattlepi/resources/usr/share/initramfs-tools/scripts/cattlepi-base/helpers

```diff
@@ -355,3 +355,60 @@ cattlepi_build_root_filesystem()
     echo $CATTLEPI_BASE > "${rootmnt}/cattlepi/base"
     echo "boot/${CATTLEPI_ID}/config" > "${rootmnt}/cattlepi/base_relative_config"
 }
+
+cattlepi_bring_up_wlan0() {
```

Comment:
Yeah, thought about that but ultimately didn't do it because we're only dealing with Rpi3.  However your suggested change will future proof us for any interface naming changes for rpi4 and beyond!

Will do.

Reply:
I've changed the name of the function and defaulted the interface to `wlan0` unless it is passed as a parameter